channels_fields/serializers.py

```diff
@@ -35,6 +34,14 @@ def to_internal_value(self, data):
         return data
 
 
+class LearningPathPreviewSerializer(serializers.ModelSerializer):
+    """Serializer for a minimal preview of Learning Paths"""
+
+    class Meta:
+        model = LearningResource
+        fields = ("title", "url", "id")
```

Comment:
In our past work on field channels, attached fully serialized learning resource objects to the field, but only ever used   `title`. In practice, the attached resources are Learning Paths, which need an extra API call anyway to get the path items.

This mini serializer helps avoid N+1 queries *and* provides the information we were using before. Since it's a subset of the full learning resource, we could always replace it with the full resources later as a non-breaking change to the API. *Though I'm not sure we will want the field APIs to be public right away anyway. They seem a bit experimental.*